spec/models/ability_spec.rb

```diff
@@ -39,245 +38,248 @@
 
   context "crop wrangling" do
 
-    before(:each) do
-      @crop = FactoryGirl.create(:crop)
-    end
+    let(:crop) { FactoryGirl.create(:crop) }
 
     context "standard member" do
       it "can't manage crops" do
-        @ability.should_not be_able_to(:update, @crop)
-        @ability.should_not be_able_to(:destroy, @crop)
+        ability.should_not be_able_to(:update, crop)
+        ability.should_not be_able_to(:destroy, crop)
       end
 
       it "can request crops" do
-        @ability.should be_able_to(:create, Crop)
+        ability.should be_able_to(:create, Crop)
       end
 
       it "can read crops" do
-        @ability.should be_able_to(:read, @crop)
+        ability.should be_able_to(:read, crop)
       end
     end
 
     context "crop wrangler" do
-      before(:each) do
-        @role = FactoryGirl.create(:crop_wrangler)
-        @member.roles << @role
-        @cw_ability = Ability.new(@member)
+
+      let(:role) { FactoryGirl.create(:crop_wrangler) }
+
+      before do
+        member.roles << role
       end
 
+      let(:cw_ability) { Ability.new(member) }
```

Comment:
Now that `ability` is a local variable and populated lazily, I think you can use it instead of `cw_ability`.